Abstract Syntax and Semantics Document/index.html

```diff
@@ -497,6 +490,107 @@
+        </p>
+
+        <p  class="note">
+          The class of time-bounded RDF streams using timestamp predicates with pairwise-bounded timescales is closed under stream merger and union.
+        </p>
+      
+      <h3>Temporal-count-bounded RDF Streams</h3>
+      
+      <p>A <em>temporal-count-bounded RDF stream</em> is an RDF stream where for each timestamp predicate that occurs in the stream, there are a finite number of temporal entities in the stream that are timestamps for that predicate.</p>
+      <p class="ednote">The qualifier "temporal" has been added to this term to emphasize that it is temporal entities that are being counted, not
+      timestamped graphs.</p>
+        <p  class="ednote">
+          Given an RDF stream whose set of timestamp predicates with pairwise-bounded temporal entities, 
+          then if the stream is temporal-count-bounded it is also time-bounded, since the upper bound for 
+          each timestamp predicate can be taken as an upper bound of the (finite) set of its timestamps, and similarly for the lower bound.
+          If the definition of "time-bounded" was changed as in the earlier Editor's Note, then every temporal-count-bounded RDF stream
+          would be time-bounded.
+        </p>
+
+        <p  class="note">
+          Every temporal-count-bounded RDF stream using only timestamp predicates with pairwise-bounded timescales is time-bounded.
+        </p>
+      
+        <p  class="note">
+          The class of temporal-count-bounded RDF streams is closed under stream merger and union, 
+          since the sets of temporal entities for each timestamp predicate in the resulting stream
+          are the union of the corresponding (finite) sets in the original streams, and so are finite. 
+        </p>
+
+        <pre  class="example highlight" title="As a pathological example, an RDF stream representing Zeno's paradox using a continuous timescale (e.g. Julian date) would be time-bounded but not temporal-count-bounded.">          
+        </pre>
+      
+    <h3>Finite RDF Stream</h3>
+    
+      <p>A <em>finite RDF stream</em> is an RDF stream of finite length, i.e. with a finite number of elements in it.</p>
+        <p  class="note">
+          Clearly, every finite RDF stream is temporal-count-bounded.
+        </p>
+        <p  class="ednote">
+          Every finite RDF stream using only pairwise-bounded timescales is time-bounded.
+          However, this is not the case for branching timescales.
+        </p>
+      
+        <p  class="note">
+          Clearly, the class of finite RDF streams is closed under stream merger and union.
+        </p>
+
+        <pre  class="example highlight" title="As another pathological example, an RDF stream that contained an infinite set of timestamped graphs all with the same timestamp predicate and  timestamp would be temporal-count-bounded but not be finite.">
```

Comment:
are these examples a motivation to change the definition or are we fine with that.